kong/db/strategies/postgres/init.lua

```diff
@@ -27,6 +28,8 @@ local log           = ngx.log
 local NOTICE        = ngx.NOTICE
 local LIMIT         = {}
 local UNIQUE        = {}
+local CREATED_AT    = "created_at"
+local TTL           = "ttl"
```

Comment:
minor: I believe we're moving away from the pattern of making variables for string constants (because if the constant is changed the variable name would have to be changed as well for consistency, so these don't gain us much).